modules/curse/curse.gd

```diff
@@ -1,45 +1,58 @@
 extends Node2D
 
 #const var
-const target_angle_epsilon = 0.1* TAU
-const line_circle_radius = 30 #pixel
-const pointerSpeed = 10 #pixel/second
-const pointerOffset = Vector2(0,-30) #pixel
-const skullSpeed = 10 #pixel/second
+const TARGET_ANGLE_EPSILON = 0.1* TAU
+const SKULL_SPEED = 10 #pixel/second
 
+const ANIM_IDLE = "Skull_Idle"
+const ANIM_INTRO = "skull_intro"
+const ANIM_OUTRO = "skull_outro"
 # editor var
 @export_node_path("Node2D") var INITIALLY_CURSED_CHARACTER : NodePath
 
 #ingame var
 var _cursed_character : Node2D = null
 var _cursable_character : Node2D = null
+var _controller_enabled : bool = false
 
-@onready var line : Node2D = get_node("CurvedLines").get_child(0)
-@onready var skullPathFollow : Node2D = get_node("Path2D/PathFollow2D")
+@onready var skull_sprite : Node2D = get_node("Path2D/PathFollow2D/Skull")
```

Comment:
On the contrary this variable should not be accessible from outside this script and should be renamed to _skull_sprite.
I would also suggest avoiding declaring those kind of variable and use the $Path2D/PathFollow2D/Skull synthax directly in the code.